src/ProjectService.Business/Commands/Task/GetTaskCommand.cs

```diff
@@ -33,27 +34,24 @@ public class GetTaskCommand : IGetTaskCommand
         private readonly ITaskResponseMapper _taskResponseMapper;
         private readonly ITaskInfoMapper _taskInfoMapper;
         private readonly ILogger<GetTaskCommand> _logger;
-        private readonly IRequestClient<IGetDepartmentRequest> _departmentRequestClient;
+        private readonly IRequestClient<IGetCompanyEmployeesRequest> _rcGetCompanyEmployee;
         private readonly IRequestClient<IGetUsersDataRequest> _usersDataRequestClient;
 
-        private IGetDepartmentResponse GetDepartment(Guid userId, List<string> errors)
+        private DepartmentData GetDepartment(Guid authorId, List<string> errors)
         {
-            string errorMessage = "Cannot get user's department. Please try again later.";
+            string errorMessage = "Cannot create task. Please try again later.";
```

Comment:
the strange message, you try to get department. Why "create task"?

Reply:
fixed